build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 ext {
     wordPressUtilsVersion = '2.5.0'
     wordPressLoginVersion = '0.14.0'
-    gutenbergMobileVersion = 'v1.78.0'
+    gutenbergMobileVersion = '4961-214827b159cbeea00415ff30727388758e656c4e'
```

Comment:
**Before merging this PR,** we'll need to update this to a new 1.78.1 release tag.